drivers/media/i2c/irs1125.c

```diff
@@ -614,8 +613,23 @@ static int irs1125_s_ctrl(struct v4l2_ctrl *ctrl)
 	}
 	case IRS1125_CID_MOD_PLL: {
 		struct irs1125_mod_pll *mod_new;
+		u8 *pval;
+		bool valid;
+		int s;
 
+		s = sizeof(struct irs1125_mod_pll) * IRS1125_NUM_MOD_PLLS;
```

Comment:
Can be done as the one line
```int s = sizeof(struct irs1125_mod_pll) * IRS1125_NUM_MOD_PLLS;```
(line wrap at the * if it's over 80 chars)

Reply:
Sure, this is changed now